README.md

```diff
@@ -35,6 +35,14 @@ To build/install the port, run:
 
 ```cd /usr/ports/net/termshark/ && make install clean```
 
+# Ubuntu
+
+You can use the PPA *nicolais/termshark* to install termshark:
+
+ * `sudo add-apt-repository ppa:nicolais/termshark`
```

Comment:
Install instructions should be a code block that includes `sudo apt update`. Language hinting is preferred (in this case, sh).

\`\`\`sh
...
\`\`\`

Reply:
I've changed the comment to use `-` instead of a `*` (as with the other lines), but I think the language hinting might only work for "full code blocks" (not single lines in a list?).